times_reader/datatypes.py

```diff
@@ -61,7 +61,7 @@ class Tag(str, Enum):
     uc_sets = "~UC_SETS"
     uc_t = "~UC_T"
     # This is used by Veda for unit conversion when displaying results
-    # unitconversion = "~UNITCONVERSION"
+    unitconversion = "~UNITCONVERSION"
```

Comment:
@olejandro is it okay to uncomment this? This is needed for the check below.

Reply:
It is definitely okay to uncomment it. I wondering how we should go about this one though, because we won't use the info provided by this type of an input table (so there is no need normalise or do anything else about it). At the same time, it is good for it to appear in `raw_tables.txt` for diffing purposes.